content/docs/04.5-devx/03-cluster-groups.md

```diff
@@ -20,18 +20,21 @@ The enterprise developer can create and utilize the Palette compute under the sc
 
 <br />
 
-* The Cluster Groups available under the Palette System Scope
+* The Cluster Groups available under the Palette [System Scope](/devx/cluster-groups#systemscope).
 
 
-* The Cluster Groups launched in the developer's enterprise datacenter/environment
+* The Cluster Groups launched in the [developer's enterprise datacenter/environment](/devx/cluster-groups#tenantscope).
 
-## Cluster Groups
+# Cluster Groups
 Cluster Group is an aggregation of host clusters deployed through Palette. Palette sandbox clusters can be launched into these host cluster groups. The cluster groups are created under two scopes:
 
-* System Scope
-* Tenant Scope
+## System Scope
 
-### Create Your Cluster Group 
+  * Palette provides a default cluster group called the Beehive Cluster group at the system scope in the `us-east` region. 
```

Comment:
@resmaa, as far as I recall, the beehive cluster group shows up in lower case on our UI, isn't that the case @tiwarisumit3?

Reply:
We could go with this as it is a console value.

```suggestion
  * Palette provides a default cluster group called `beehive` at the system scope in the us-east region. 
```